specs/airr-schema.yaml

```diff
@@ -1954,24 +1973,60 @@ Clone:
         deletions:
             type:  string
             description: Clone deletion mutations
+        CloneRepertoireList:
```

Comment:
is using `List` the standard in AIRR schemas or would it be acceptable to just pluralize it into `CloneRepertoires`?

Reply:
dunno - @schristley and @javh ?